source/refills/tooltip.html.erb

```diff
@@ -0,0 +1,3 @@
+<a href="#0" class="tooltipped" aria-label="Tooltip description">
```

Comment:
Yeah, that's how I originally went about implementing, but now realize that the text link is a  poor example with the changes in the last couple commits. I think this is more applicable (with current implementation) for graphical buttons and links, like in GitHub's UI.

I also think that's more practical for real life usage. So, I'm going to update the link text to an svg or something.